test/functional/desired.js

```diff
@@ -87,6 +88,7 @@ const MULTIPLE_APPS = amendCapabilities(GENERIC_CAPS, {
 
 const TOUCHIDAPP_CAPS = amendCapabilities(GENERIC_CAPS, {
   'appium:app': APPS.touchIdApp,
+  'appium:deviceName': 'iPhone 8'
```

Comment:
is it necessary to hardcode a specific device name here?

Reply:
yes, I prefer to use a device which supports touch id authentication.

Reply:
I mean it is probably not a good idea to hardcode it here. Can we just reuse the one supplied with DEVICE_NAME?

Reply:
> I mean it is probably not a good idea to hardcode it here. Can we just reuse the one supplied with DEVICE_NAME?

I'm afraid but I don't think it works on CI because it uses iPhone X (which does not support touch ID) as far as checking https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=24438&view=logs&j=8563cc7c-03aa-53b4-5f4c-ac3dae66949f&t=2a36cd2c-6ce5-5d2b-8542-870a64b5be7c. 

So, if we respect DEVICE_NAME for this tests, we need to update this tests. For example, we need to change `should accept matching fingerprint if touchID is enrolled` test like the following.
```
it('should accept matching fingerprint if touchID is enrolled', async function () {
  const authenticateButton = await driver.$('~Authenticate with Touch ID');
  await authenticateButton.click();
  await B.delay(3000);
  
  const touchIdDailog = driver.$(TOUCH_ID_LOCATOR);
  if (touchIdDailog.elementId) { // when given a device which supports touch ID
    await waitUntilExist(TOUCH_ID_LOCATOR);
    await driver.execute('mobile: sendBiometricMatch', {type: 'touchId', match: true});
    expect(await driver.$('~Succeeded').elementId).to.exist;
  } else { // For devices which doesn't have a touch ID like iPhone X
    expect(await driver.$('~Touch ID not supported').elementId).to.exist;
  }  
});
```

However, we may overlook a defect on AUT or `mobile: sendBiometricMatch` api. If it is ok for us, I'll update.

Reply:
then lets add another env constant for device name supporting touch id or implement face id instead for devices that support it (iPhone 8 will be deprecated anyway sooner or later)

Reply:
thank you for your advice. added `DEVICE_NAME_FOR_TOUCH_ID` constant. as for face id, I'll make another PR.